package.json

```diff
@@ -62,6 +62,7 @@
     "istanbul": "^1.1.0-alpha.1",
     "jsdoc": "^3.5.5",
     "mocha": "^5.2.0",
+    "mockdate": "^2.0.2",
```

Comment:
I like how `mockdate` has no dependencies itself, but adding dependencies to this project will make it very difficult to have your PR accepted. Would it be easy to write the test without adding the dependency?